changelog.md

```diff
@@ -1,65 +1,68 @@
 # Changelog
 
-## v1.0.0 | t.b.d.
+## v1.0.0 | 2023-03-14
+
+Happy π-day!
+
+#### New features
+
+* Added links to example data for all [tutorials](https://lumicks-pylake.readthedocs.io/en/v1.0.0/tutorial/index.html). Try running [them](https://lumicks-pylake.readthedocs.io/en/v1.0.0/tutorial/index.html) in a jupyter notebook!
```

Comment:
Maybe just the one link, I found myself trying to figure out what the difference between the two was

```suggestion
* Added links to example data for all [tutorials](https://lumicks-pylake.readthedocs.io/en/v1.0.0/tutorial/index.html). Try running them in a `jupyter` notebook!
```